README.md

```diff
@@ -112,23 +112,31 @@ In either case, you will want to have [Serverless] installed, eg. `npm install -
   - serverless-haskell
   ```
 
-* Use `sls deploy` to deploy the executable to AWS Lambda.
+* Build and test locally using `sls invoke local`
 
-  The `serverless-haskell` plugin will build the package using Stack and upload
-  it to AWS together with a JavaScript wrapper to pass the input and output
-  from/to AWS Lambda.
+  The `serverless-haskell` plugin will build the package using Stack. Note that the first build can take around. Consider adding `export SLS_DEBUG=*` so you can see what is happening.
```

Comment:
The first build can take around ... ?

I would just say "the first build can take a long time."